packages/flutter_tools/templates/ffi_plugin/pubspec.yaml.tmpl

```diff
@@ -0,0 +1,90 @@
+name: {{projectName}}
+description: {{description}}
+version: 0.0.1
+homepage:
+
+environment:
+  sdk: {{dartSdkVersionBounds}}
+  flutter: ">=2.5.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+  flutter_lints: ^1.0.0
+
+# For information on the generic Dart part of this file, see the
+# following page: https://dart.dev/tools/pub/pubspec
+
+# The following section is specific to Flutter.
+flutter:
+  # This section identifies this Flutter project as a plugin project.
+  # The 'pluginClass', ffiPlugin, and Android 'package' identifiers should not ordinarily
+  # be modified. They are used by the tooling to maintain consistency when
+  # adding or updating assets for this project.
+  plugin:
+    platforms:
+{{#no_platforms}}
+    # This FFI plugin project was generated without specifying any
+    # platforms with the `--platform` argument. If you see the `some_platform` map below, remove it and
+    # then add platforms following the instruction here:
+    # https://flutter.dev/docs/development/packages-and-plugins/developing-packages#plugin-platforms
+    # -------------------
+      some_platform:
+        ffiPlugin: true
+    # -------------------
+{{/no_platforms}}
+{{#android}}
+      android:
+        ffiPlugin: true
+{{/android}}
+{{#ios}}
+      ios:
+        ffiPlugin: true
+{{/ios}}
+{{#linux}}
+      linux:
+        ffiPlugin: true
+{{/linux}}
+{{#macos}}
+      macos:
+        ffiPlugin: true
+{{/macos}}
+{{#windows}}
+      windows:
+        ffiPlugin: true
+{{/windows}}
```

Comment:
this file should explain what `ffiPlugin` means, what the implications are, why you would use it, how you would migrate to it or away from it, etc.